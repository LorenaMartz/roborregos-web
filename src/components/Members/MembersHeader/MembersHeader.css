.members-header {
  height: 100%;
  position: relative;
  background: linear-gradient( 45deg, rgba(20, 18, 19, 0.9) 10%, rgba(0, 94, 105, 0.9) 100%), url(../../../images/github_pattern.svg) top left/auto repeat fixed;
  color: rgba(255, 255, 255, 0.8);
}

.members-header .main-text-members {
  color: rgba(255, 255, 255, 0.9);
  font-size: 3vh;
  text-transform: none;
  padding-top: 2%;
  padding-bottom: 0%;
}

.title-text-banner {
  color: rgba(255, 255, 255, 0.9);
  font-size: 7vh;
  text-transform: none;
}

.members-header .members-header-footer {
  font-size: 2.5vh;
  text-transform: none;
  display: flex;
  flex-direction: column;
  align-items: center;
  text-align: center;
  position: absolute;
  bottom: 5vh;
  left: 0;
  right: 0;
  margin-left:auto;
  margin-right:auto;
}

.members-header .members-header-footer p {
  margin: 0;
}

.members-header .members-header-footer .icon-btn {
  cursor: pointer;
  color: rgb(0, 225, 250);
  font-size: 30px;
}

.members-header .members-header-footer .icon-btn:hover {
  color: rgb(196, 63, 101);
}
