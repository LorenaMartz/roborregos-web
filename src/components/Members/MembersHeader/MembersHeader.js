import React, { Component } from 'react';
import './MembersHeader.css';

class MembersHeader extends Component {
	render() {
		return(
			<div className='members-header'>
				<div className='container-legend'>
					<h2>
						Miembros
					</h2>
				</div>
			</div>
		);
	}
}

export default MembersHeader;