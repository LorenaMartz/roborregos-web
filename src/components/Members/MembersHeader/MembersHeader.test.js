// @flow
import { getByTestId } from '@testing-library/react'
import React from 'react'
import { render, unmountComponentAtNode } from 'react-dom'
import { act } from 'react-dom/test-utils'
import renderer from 'react-test-renderer'

import MembersHeader from './MembersHeader'

let container = null
beforeEach(() => {
  // setup a DOM element as a render target
  container = document.createElement('div')
  if (document.body != null) {
    document.body.appendChild(container)
  } else {
    expect(document.body).not.toEqual(null)
  }
})

afterEach(() => {
  // cleanup on exiting
  unmountComponentAtNode(container)
  if (container != null) {
    container.remove()
  } else {
    expect(container).not.toEqual(null)
  }
  container = null
})

it('MemberÂ´s Join Us section renders correctly', () => {
  act(() => {
    render(<MembersHeader />, container)
  })
  expect(getByTestId(container, 'members-header')).not.toEqual(null)
  expect(getByTestId(container, 'members-header-title')).not.toEqual(null)
  expect(getByTestId(container, 'members-header-text')).not.toEqual(null)
})

it('Matches snapshot', () => {
  const tree = renderer.create(<MembersHeader />).toJSON()
  expect(tree).toMatchSnapshot()
})
