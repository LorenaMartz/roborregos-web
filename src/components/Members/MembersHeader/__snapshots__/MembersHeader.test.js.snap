// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshot 1`] = `
<div
  className="members-header"
  data-testid="members-header"
>
  <div
    className="container-legend"
  >
    <h2
      className="title-text-banner"
      data-testid="members-header-title"
    >
      Members
    </h2>
    <div
      className="main-text-members"
      data-testid="members-header-text"
    >
      <p>
        RoBorregoâ€™s community is made by students with different skills in robotics, logistics and networking, all joined with a passion for exploring new technologies and sharing their knowledge with everybody.
      </p>
    </div>
  </div>
  <div
    className="members-header-footer"
  >
    <p>
      Scroll down and meet the us!
    </p>
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-angle-down fa-w-10 icon-btn"
      data-icon="angle-down"
      data-prefix="fas"
      focusable="false"
      onClick={[Function]}
      role="img"
      style={Object {}}
      viewBox="0 0 320 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M143 352.3L7 216.3c-9.4-9.4-9.4-24.6 0-33.9l22.6-22.6c9.4-9.4 24.6-9.4 33.9 0l96.4 96.4 96.4-96.4c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9l-136 136c-9.2 9.4-24.4 9.4-33.8 0z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
  </div>
</div>
`;
